pages/over-ori-a.md

```diff
@@ -9,25 +9,29 @@ Hieronder wordt het belang van een archiefstandaard voor zulke informatie uitgel
 
 # Wat is ORI-A?
 
-De Open Raadsinformatie Archiefstandaard (ORI-A) beschrijft de regels voor het duurzaam bewaren en uitwisselen van raadsinformatie in XML-formaat. Hiervoor is het ORI-A XML-schema ontwikkeld, dat kan worden gebruikt wanneer raadsinformatie, zoals een collectie videotulen, voor permanente bewaring naar een [e-depot](https://www.nationaalarchief.nl/archiveren/kennisbank/wat-is-een-e-depot) wordt gemigreerd. 
+De Open Raadsinformatie Archiefstandaard (ORI-A) beschrijft de regels voor het duurzaam bewaren van raadsinformatie in XML-formaat. Deze regels zijn vastgelegd in het [ORI-A XML-schema](downloads#xml-schema). Dit schema kan gebruikt worden wanneer raadsinformatie, zoals een collectie videotulen, voor permanente bewaring naar een [e-depot](https://www.nationaalarchief.nl/archiveren/kennisbank/wat-is-een-e-depot) wordt gemigreerd.
 
-ORI-A is gebaseerd op het door de VNG beheerde informatiemodel van [de Open Raadsinformatie (ORI) API Specificatie](https://github.com/VNG-Realisatie/ODS-Open-Raadsinformatie).
+ORI-A is gebaseerd op het informatiemodel dat de VNG heeft ontworpen voor [de Open Raadsinformatie (ORI) API](https://github.com/VNG-Realisatie/ODS-Open-Raadsinformatie). In ["Hoe verhoudt ORI-A zich tot ORI?"](faq) kun je meer lezen over de verschillen tussen deze twee informatiemodellen.
```

Comment:
Deze zinssnede komt straight uit de FAQ, en is korter. Een API is al een specificatie, dus dat woord voegde weinig toe. De toevoegde zin hakt er nog eens in dat er twee standaarden/informatiemodellen zijn. Zonder deze toevoeging vind ik dat deze inleidende pagina daar te weinig nadruk op legt (een mention), en te weinig informatie over biedt (nu weten mensen die de verschillen willen bestuderen waar ze moeten kijken)